ui/src/AuthService.js

```diff
@@ -0,0 +1,55 @@
+import config from './config/config';
+
+/**
+ * Creates a configuration obect with information about the request type
+ * and the request body
+ *
+ * @param {string} method - HTTP Method (PUT, GET, PATCH, DELETE, POST)
+ * @param {object|Array} body - JSON Payload to send in the request
+ * @returns a config object to send in HTTP requests
+ */
+const requestConfig = (method, body) => ({
```

Comment:
Out of interest what is the motivation behind having a separate service for auth? Is it because it's calling to oauth rather than to our backend service?

Reply:
Yes because it is calling a completely separate service. The DataService is coupled to the API.

Reply:
> Code looks good @craicoverflow - good work 
> 
> I haven't verified as @laurafitzgerald has already done so.
> 
> Only comment I would make is with 2 services now using retrieving data with fetch maybe we could abstract it out to a seperate http layer, but that may be unnecessary at this point.

I agree, though the `request` function in each are slightly different as the structure of the data returned from each are different. Our API returns a JSON object on error whereas oauth returns a string.